### PR TITLE
Update github templates to reflect stories and epics

### PR DESCRIPTION
**Pull request by colinmurphy01**

### CODE REVIEW

**Comment by colinmurphy01** [comment]
@mgwalker can we update the project board logic to **not** automatically pull in tickets with the "⛰️ Epic" tag. And instead have them go to [the Roadmap board](https://github.com/orgs/18F/projects/52/views/1)?

**Comment by mgwalker** [comment]
I believe that works. It looks like the automation takes a filter, which should also let us do something like `-label:⛰️ Epic`

**Comment by mgwalker** [comment]
Yep, we can do that. And it's done now.

File: .github/ISSUE_TEMPLATE/story-template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+about: Use this template to create a story and track in our sprint board.
+title: ''
+labels: 'Feature, :rock: Story'
+assignees: ''
+
+---
+
+### Description
+
+**Related Boulder: [Epic]**
```
Comment (mgwalker) [inline .github/ISSUE_TEMPLATE/story-template.md:12]:
```suggestion
**Related Epic: [#Epic]**
```

**Review by mgwalker — APPROVED** [review]
Suggested changing `related boulder` to `related epic`, but this is good to go whenever!

**Pull request merged**